src/services/ckan/datasetList.ts

```diff
@@ -23,19 +24,26 @@ export const makeDatasetList = (DMS: string) => {
 };
 
 const buildFilterQueryPart = (filters: string[]): string => {
-  return filters.length > 0 ? `(${filters.join(' OR ')})` : '';
+  return filters.length > 0
+    ? `(${filters
+        .map((filter: string) => (filter.startsWith('http') ? `"${filter}"` : filter))
```

Comment:
Unit test for this new logic. Let's add a separate ticket in backlog for adding tests for all this logic

This ticket can be picked in future

Reply:
Unit test has now been implemented